ast27/Include/asdl.h

```diff
@@ -28,11 +28,11 @@ typedef struct {
 } asdl_int_seq;
 
 #if PY_MINOR_VERSION > 3
-#define asdl_seq_new _Py_asdl_seq_new
-#define asdl_int_seq_new _Py_asdl_int_seq_new
+#define asdl_seq_new _Ta27_asdl_seq_new
+#define asdl_int_seq_new _Ta27_asdl_int_seq_new
 #else
-#define _Py_asdl_seq_new asdl_seq_new
-#define _Py_asdl_int_seq_new asdl_int_seq_new
+#define _Ta27_asdl_seq_new asdl_seq_new
+#define _Ta27_asdl_int_seq_new asdl_int_seq_new
 #endif
```

Comment:
Why even have the if/else here? We don't support Python 3.3 or before, do we?

Reply:
Nope, I was just making a commit to remove this!